src/core/qgssettings.h

```diff
@@ -182,11 +182,13 @@ class CORE_EXPORT QgsSettings : public QObject
     void remove( const QString &key );
     //! Return the sanitized and prefixed key
     QString prefixedKey( const QString &key, const Section section ) const;
+    //! Removes all entries in the primary location associated to this QgsSettings object.
```

Comment:
For consistency, we've been calling the "primary location" "User Settings", the read-only system-wide settings are called "Global Settings".